poetry.lock

```diff
@@ -108,10 +108,10 @@ python-versions = ">=3.5"
 dev = ["cloudpickle", "coverage[toml] (>=5.0.2)", "furo", "hypothesis", "mypy (>=0.900,!=0.940)", "pre-commit", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "sphinx", "sphinx-notfound-page", "zope.interface"]
```

Comment:
I'm assuming that this file is auto-generated and doesn't need my review, right?

Reply:
yep it is autogenerated